src/pages/Clients/index.js

```diff
@@ -0,0 +1,242 @@
+import React, { useState, useEffect } from "react";
+import axios from "../../config/API";
+import { animateScroll as scroll } from "react-scroll";
+import { getAuthorizationHeader } from "../../services/Authentication";
+import { Formik } from "formik";
+import FormMultipleSelect from "../../components/FormComponents/FormMultipleSelect";
+import {
+  realEstateTypes,
+  neighborhoods,
+} from "../../utils/FormHelpers/form-data";
+import { formatCollectionForReactSelect } from "../../utils/FormHelpers/FormatCollectionForReactSelect";
+import { FormTextField } from "../../components/FormComponents/FormTextField";
+import parseJwt from "../../utils/parseJwt";
+
+function AddClient({
+  setNewClientData,
+  newClientData,
+  setShowForm,
+  clientsData,
+  setClientsData,
+}) {
+  const handlePost = async (values) => {
```

Comment:
Event handler function names should point to the event 
```suggestion
  const handleSubmit= async (values) => {
```

Reply:
Noted. The reason behind naming the function different than handleSubmit was related to that I did find it duplicating to name it that way after we already have a handleSubmit coming from Formik.